source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,115 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to synchronize data from a replica set to a 
+sharded cluster using :program:`mongosync`. 
+
+.. note::
+
+   ``mongosync`` doesn't support sync from a sharded cluster to a replica set.
+
+To perform a sync between two replica sets, see :ref:`c2c-quickstart`.
+
+Before You Begin
+----------------
+
+Before you begin your sync:
+
+- Define your source replica set.
+- Define your destination sharded cluster.
+- Define an administrative user that has permissions in both clusters.
+
+You must also download ``mongosync``. To learn more, see 
+the :ref:`mongosync setup <c2c-quickstart-setup>` instructions.
+
+Steps 
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Connect the replica set and sharded cluster. 
+
+      ``mongosync`` must create an initial connection between the source 
+      replica set and the destination sharded cluster before you can start to 
+      sync data. 
+
+      To connect ``cluster0`` to ``cluster1`` with ``mongosync``, enter the
+      following command on one line:
```

Comment:
I might remove the on one line here, its slightly confusing because the code example has `\` on multiple lines. If relevant, you can add a bullet in the AtT section to the effect:

```
- mongosync commands must be entered on a single line or have line terminated slashes (`\`). 
```
```suggestion
      following command:
```